packages/data-stores/src/site/actions.ts

```diff
@@ -285,7 +285,7 @@ export function createActions( clientCreds: WpcomClientCredentials ) {
 		yield saveSiteSettings( siteId, { blogdescription } );
 	}
 
-	function* installTheme( siteSlugOrId: string, themeSlug: string ) {
+	function* installTheme( siteSlugOrId: string | string, themeSlug: string ) {
```

Comment:
I think `string | string` is the same as `string`, and I'm not seeing any typescript errors with the original `string`. Is this doing something I'm not realizing?

Reply:
It was supposed to be `string | number` 🤦‍♂️